examples/30_extended/datasets_tutorial.py

```diff
@@ -146,4 +146,13 @@
 data_id = edit_dataset(564, default_target_attribute="y")
 print(f"Edited dataset ID: {data_id}")
 
+
+############################################################################
+# Fork dataset
+# Used to create a copy of the dataset with a different owner
```

Comment:
I think it would be good to give more details what this means - one can read up in the PR implementing this for the server - but maybe a glossary somewhere under `doc.openml.org` would be great.

Reply:
Yes, I will add this to the documentation. Especially the API itself is not documented, except in the examples in python API.

Reply:
I think I would be more explicit, e.g. prefer "Used to create a copy of the dataset with ~a different~ you as the owner".

Reply:
I agree that the general description of what forking a dataset entails and why you would do it should be hosted in the cross-platform documentation. However, I would still dedicate one or two sentences to recap it here as well (best to assume users are lazy).

Reply:
Thanks for updating the description. What I'm wondering though is, how is the dataset then finalized (i.e. critical fields should not be editable at all, so there must be a way of finalizing the dataset such that even the fork cannot be changed)? Could you please extend the example by that?

Reply:
The dataset is "finalized" when a task is created for it. After a task has been created, critical fields can no longer be edited, not even by the dataset owner. (But yes, it is a good idea to write this down in the documentation)

Reply:
Thanks @PGijsbers for the explanation. @sahithyaravi1493 could you please add that to the docs?